src/api/spec/factories/users.rb

```diff
@@ -2,7 +2,7 @@
   factory :user do
     email { Faker::Internet.email }
     realname { Faker::Name.name }
-    login { Faker::Internet.user_name(nil, %w(_)) }
+    sequence(:login){|n| "#{Faker::Internet.user_name(nil, %w(_))}#{n}" }
```

Comment:
To make login unique! From faker gem readme

`
NOTE: While Faker generates data at random, returned values are not guaranteed to be unique.
`